src/test/java/core/basesyntax/service/ShopServiceTest.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitInStorage;
+import core.basesyntax.service.impl.ShopServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class ShopServiceTest {
+    private static final ShopService SHOP_SERVICE = new ShopServiceImpl();
+
+    @AfterEach
+    void tearDown() {
+        Storage.FRUITS.clear();
+    }
+
+    @Test
+    void addFruits_addNewFruit_Ok() {
+        FruitInStorage actualFruitInStorage = SHOP_SERVICE.addFruits("banana", 12);
+        FruitInStorage expectedFruitInStorage = new FruitInStorage("banana", 12);
+        assertEquals(expectedFruitInStorage, actualFruitInStorage);
+    }
+
+    @Test
+    void addFruits_addExistingFruit_Ok() {
+        FruitInStorage actualFruitInStorage = SHOP_SERVICE.addFruits("banana", 12);
```

Comment:
If you want to test adding existing fruit you should add it in another way (add to the collection directly)
It’s  bad to use the same method that you test for data preparation  
check it in other test cases